alphastats/plots/DimensionalityReduction.py

```diff
@@ -119,7 +120,7 @@ def _pca(self):
         }
 
     def _tsne(self, **kwargs):
-        tsne = sklearn.manifold.TSNE(n_components=2, verbose=1, **kwargs)
```

Comment:
why? :)

why not go from `sklearn.decomposition.PCA` to `PCA` too in that case? (just a question, I actually like verbose init because it's instantly clear from which package the class comes)

Reply:
agree, will revert to verbose import in this case